src/probe/sockets-location-filter.ts

```diff
@@ -1,153 +0,0 @@
-import config from 'config';
```

Comment:
Small note here - when renaming files, it's really good to commit the rename on its own and then make any changes separately so that git diff understands it's the same file and doesn't show it as one full delete and second full add (which is hard to review).